### DISCUSSION TITLE
Langflow newest Version locally

### DISCUSSION THREAD

**Discussion by japa-le** [langflow-ai/langflow#673] [Q&A]
Hello, when I install Langflow locally I just get the 0.2.7 version. How can I get the newest version?

**Comment by kyuumeitai** [comment]
Hi, you can try to install langchain and force the version.

`pip install langflow==0.3.3
`

Once you do this, you either install the 0.3.3 or it will give the reason for the 0.2.7, in my case, I was using python 3.11.4 in my conda setup, and langflow requires minor 3.11. Creating another conda environment with 3.10 solves my problem. Hope it helps!

**Reply by TFWol** [reply]
I hit the same issue and ended up doing pretty much the same.
```batch
conda create -n langflow python==3.10
conda activate langflow
pip install --force-reinstall langflow[local]==0.4.16
```

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
